test/com/google/enterprise/adaptor/fs/FsAdaptorTest.java

```diff
@@ -280,6 +280,7 @@ public void testAdaptorInitDfsNamespaceWithBadDfsLink() throws Exception {
 
   private void makeDfsNamespace(MockFile dfsNamespace) {
     dfsNamespace.setIsDfsNamespace(true);
+    dfsNamespace.setDfsShareAclView(MockFile.FULL_ACCESS_ACLVIEW);
```

Comment:
It seems like this value is never verified in the output, just needed so readShareAcls doesn't blow up?